github/orgs_teams.go

```diff
@@ -44,13 +44,15 @@ func (t Team) String() string {
 	return Stringify(t)
 }
 
-// Invitation represents a team member's inviation status
+// Invitation represents a team member's invitation status.
 type Invitation struct {
-	ID        *int       `json:"id,omitempty"`
-	Login     *string    `json:"login,omitempty"`
-	Email     *string    `json:"email,omitempty"`
+	ID    *int    `json:"id,omitempty"`
+	Login *string `json:"login,omitempty"`
+	Email *string `json:"email,omitempty"`
+	// Role can be one of the values - direct_member, admin, billing_manager, hiring_manager, or reinstate.
```

Comment:
nit - would you mind putting the acceptable values in quotes to follow the style of the other documentation in the repo?